_posts/2020-08-26-#19339.md

```diff
@@ -223,7 +223,7 @@ commit: 67bb714
 19:23 <jnewbery> it's impossible to broadcast something that isn't in your mempool
 19:23 <robot-dreams> unlike testmempoolaccept, sendrawtransaction calls into `BroadcastTransaction`, which (i) calls ATMP a second time with `test_accept = true` and (ii) does some extra bookkeeping
 19:23 <dhruvmehta> test_accept=1 is a dry run. It runs policy checks and signature checks which are cached though so a second "actual" run is significantly cheaper.
-19:23 <gzhao408> for reference testmempoolaccept and sendrawtransaction can be found here: https://github.com/bitcoin/bitcoin/blob/master/src/rpc/rawtransaction.cpp#L800
+19:23 <gzhao408> for reference testmempoolaccept and sendrawtransaction can be found here: https://github.com/bitcoin-core-review-club/bitcoin/blob/master/src/rpc/rawtransaction.cpp#L772 and https://github.com/bitcoin-core-review-club/bitcoin/blob/master/src/rpc/rawtransaction.cpp#L833
```

Comment:
This should be changed to point to a commit hash in master in bitcoin/bitcoin

Reply:
done